src/module/wallet/component/core/WalletsBackupModal/WalletQuizBackup/WalletQuizBackup.tsx

```diff
@@ -0,0 +1,91 @@
+import { useState, useEffect } from "react";
+import { config } from "config";
+import WalletQuizBackupError from "./steps/WalletQuizBackupError";
+import WalletQuizBackupWelcome from "./steps/WalletQuizBackupWelcome";
+import { QuizQuestion, QuizStep, WalletQuizBackupProps } from "./WalletQuizBackup.stypes";
+import WalletQuizBackupQuiz from "./steps/WalletQuizBackupQuiz";
+import useTranslate from "module/common/hook/useTranslate";
+import ConfirmPinModal from "module/settings/components/core/ConfirmPinModal/ConfirmPinModal";
+import { TabPanel, Tabs } from "@peersyst/react-native-components";
+import { NotificationFeedbackType, notificationAsync } from "expo-haptics";
+
+const WalletQuizBackup = ({ onClose, onSubmit }: WalletQuizBackupProps): JSX.Element => {
+    const translateError = useTranslate("error");
+    const [quizQuestions, setQuizQuestions] = useState<QuizQuestion[]>([]);
+    const [currentQuestionIndex, setCurrentQuestionIndex] = useState<number>(0);
+    const [answers, setAnswers] = useState<number[]>([]);
+    const [error, setError] = useState<string>("");
+    const [quizStep, setQuizStep] = useState<QuizStep>(QuizStep.Welcome);
+    const [showModal, setShowModal] = useState(false);
+    const [quizQuestionsShuffled, setQuizQuestionsShuffled] = useState<QuizQuestion[]>([]);
+
+    useEffect(() => {
+        const shuffledQuestions = [...config.securityQuizList].sort(() => 0.5 - Math.random());
+        const originalShuffledQuestions = shuffledQuestions.slice(0, 3);
+        setQuizQuestions(originalShuffledQuestions);
+        const shuffledQuestionsWithShuffledResponses = originalShuffledQuestions.map((question) => {
+            const shuffledResponses = [...question.response].sort(() => 0.5 - Math.random());
+            return { ...question, response: shuffledResponses };
+        });
+
+        setQuizQuestionsShuffled(shuffledQuestionsWithShuffledResponses);
```

Comment:
No es necesario este useEffect inicial. Es mejor practica inicializar el useState directamente
https://dev.to/nibble/lazy-state-initialization-in-functional-components-1ib0